dbservice/postgresdb/dbservice_test.go

```diff
@@ -639,3 +639,113 @@ func TestInsertInTableName(t *testing.T) {
 		t.Errorf("Unexpected error in 'insertInTableName', expected: %v, got: %v", badUpdateError, err)
 	}
 }
+
+func TestInsertCfgCacheSource(t *testing.T) {
+	t.Log("happy insert")
+	savedPsqlConnect := psqlConnect
+	psqlConnect = func(connectUrl string) (*sqlx.DB, error) {
```

Comment:
could you make it parametrized and reuse `psqlConnect` code - meaning part of test data can be a function for sql config

Reply:
done